app/assets/stylesheets/moneyhelper_styles.scss

```diff
@@ -0,0 +1,445 @@
+/*
+Stylesheet to override MAS styles with Moneyhelper styles
+Fonts come from dough/assets/fonts
+ */
+
+@import "moneyhelper_variables";
+
+//Media Query Variables (copied from dough/assets/stylesheets/lib/_mediaqueries.scss - we will need to find a global
+//place for these or write to 3 into 1 mq
+
+$mq-xs: 0 !default;
+$mq-xs-max: (479px) !default;
+$mq-s: (480px) !default;
+$mq-s-max: (719px) !default;
+$mq-m: (720px) !default;
+$mq-m-max: (959px) !default;
+$mq-l: (960px) !default;
+$mq-l-max: (1199px) !default;
+$mq-xl: (1200px) !default;
+
+//Print Styles for last page button
+@media print {
+  // hide everything
+  body * {
+    visibility: hidden;
+  }
+  // show only results
+  #stewd_parent,
+  #stewd_parent * {
+    visibility: visible;
+  }
+  // show collapsible info
+  #stewd_parent .collapsable__target {
+    display: block;
+  }
+  // position the element in page
+  #stewd_parent {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+}
+
+.l-container-tool {
```

Comment:
All styles between lines 44 - 445 are nested within a class (`.l-container-tool`) which doesn't exist and are therefore not being used. These should be deleted.